tests/test_harness.py

```diff
@@ -212,17 +213,18 @@ def test_live_logging(
         oss_artifact: Path,
     ) -> None:
         """
-        If `log_output_live` is given as `True`, subprocess output is logged.
+        If `log_output_live` is given as `True`, the installation output is
```

Comment:
I think that this test is now in the wrong place. This is:

```
class TestClusterLogging:
    """
    Tests for logs created by the ``Cluster``.
    """
```

Perhaps this should be in a `TestInstallDcosFromPath` class?